snippets/hashNode.md

```diff
@@ -0,0 +1,14 @@
+### hashNode
+
+Creates a hash for a value using the [SHA-256](https://en.wikipedia.org/wiki/SHA-2) algorithm.
+
+Use `crypto` API to create a hash for the given value.
+
+```js
+const crypto = require('crypto');
```

Comment:
# Question
In node based snippets should they have the require statement in them un-commented?

Reply:
I was thinking that we could actually have the require statement inside the function itself. I think that can work and de-clutter the global namespace, right?

Reply:
I will quickly submit a PR that will parse handle require. @skatcat31 I honestly think leaving it there is ok. Just need to be able parse it & handle it accordingly

Reply:
@Chalarangelo do NOT put require statements in functions. It will lead to a lot of overhead that in the worse case scenario could lead to disk access if the module has been fully garbage collected and de-instanced between each use. A require statement in node actually hits a cache algorithm that is very slow. Hence why requires should only be done once and kept around.

If the module in node has been designed badly it can lead to a worse case scenario of a memory leak. DRY coding practices should be adhered to as much as possible. Some good resources on the subject include Vitaly's write up on it in pg-promise and the bluebirdjs write up on it. More information can also be found in the node best practices repo, etc.

Requires and imports are almost analogous, so same applies there

Reply:
@skatcat31 Had no idea, thanks a ton for clarifying this! I suggest we keep them as they are, then, commenting them provides no significant benefit imho.

Reply:
@Chalarangelo fair, since it does hit the caching algorithm it should reuse. the problem comes in ones that requires instances